### PR TITLE
Account settings view (change password, manage ssh keys)

### PR DESCRIPTION
**Pull request by sonph**
untitled

GIF ftw

### CODE REVIEW

**Comment by coffeefee-coder** [comment]
+1

**Comment by Spencatro** [comment]
looks quite good +1

File: assets/js/app.js [hunk 1]
```diff
@@ -282,7 +282,48 @@ app.controller('SettingsCtrl', function($scope, $rootScope, $http) {
     title: 'Account Settings'
   }
 
+  $scope.keys = [
+    {
```
Comment (Spencatro) [inline assets/js/app.js:286]: In the future, this section will be empty, right?

Comment (sonph) [inline assets/js/app.js:286]: Yeah the data will be from the server

**Pull request merged**